test/graphql/types/Mutation/createChatMembership.test.ts

```
@@ -0,0 +1,604 @@
+	type Mock,
+	afterEach,
+	beforeEach,
+	describe,
+	expect,
+	it,
+	vi,
+} from "vitest";
+import { TalawaGraphQLError } from "~/src/utilities/TalawaGraphQLError";
+import { ChatMembershipResolver } from "../../../../src/graphql/types/Mutation/createChatMembership";
+
+type Fields = { [key: string]: any }; // Replace with actual field types
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)